node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
+try {
+  // obtaining the file list using array destructuring
```

Comment:
Don't write comments that just explain what the code is doing. I can read the code to figure that out. Your comments should be explaining _why_, not _how_. You want comments when the code is not obvious because you are doing something tricky, or fixing a challenging bug, or explaining why you chose a particular way to implement.